addons/web/static/src/views/form/form_renderer.js

```diff
@@ -87,6 +87,9 @@ FormRenderer.components = {
     InnerGroup,
     StatusBarButtons,
 };
+FormRenderer.props = {
+    "*": true,
+};
```

Comment:
fine for this PR, but for this component I think that we could have something more precise